css/main.css

```diff
@@ -25,10 +25,76 @@ header {
   padding: 15px;
   background-color: #3c3a39;
   color: white;
+  position: fixed;
+  z-index: 1;
+  width: 100vw;
 }
 
-#desktop-nav, #desktop-nav ul {
-  display: none;
+.navbar {
+  width: 200px;
+  height: 100%;
+  background-color: #262626;
+  position: fixed;
+  top: 0;
+  right: -300px;
+  display: flex;
```

Comment:
### OPTIONAL
- [ ] For DRY CSS you should combine duplicate properties. Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/)